### PR TITLE
Use Danger swift

### PR DESCRIPTION
**Pull request by orta**
I'm a tad wary of the danger in `post` ( it might mean the build can't be failed ) on CI, but this moves everything ~~but Swift Lint checks~~ into Danger Swift.

You could probably write a quick `JSON -> warns` thing in the bottom of the file for danger-swift, like the old v1 version of the ruby plugin.

### CODE REVIEW

**Comment by orta** [comment]
hah - that fail message

**Comment by orta** [comment]
OK, that's also got a simple version of Danger Swiftlint in it - ready to go.

**Comment by ashfurrow** [comment]
Thanks @orta!

File: circle.yml [hunk 1]
```diff
@@ -3,16 +3,18 @@ machine:
     version: 9.0
   environment:
     LANG: en_US.UTF-8
+    HOMEBREW_NO_AUTO_UPDATE: 1
 
 dependencies:
   override:
   - brew install swiftlint
+  - npm install -g danger@alpha
```
Comment (orta) [inline circle.yml]: For anyone using this as a reference, you should use `npm install -g danger` instead now that [Danger JS 2.0](https://spectrum.chat/danger/announcements?thread=8a9c08c5-bafe-4272-b41e-f6bb34fc1b45) is out 👍

**Pull request merged**